proto/common.proto

```diff
@@ -75,6 +75,23 @@ message WorkerNode {
   // It's populated by meta node, when the worker node is added by meta node.
   // It's not persistent in meta store.
   optional uint64 started_at = 9;
+
+ 
+  optional string label = 10;
+
+  // Meta may assign labels to worker nodes to partition workload by label.
+  // This is used for serverless backfilling of MVs. 
+  // If a WorkerNode has labels set it indicates that this node only accepts certain work.
+  // If a WorkerNode has no labels set it indicates that this node only any work.
```

Comment:
@shanicky Are you ok with this? We can also be more explicit and enforce that `node_labels` cannot be empty.